### DISCUSSION TITLE
Hires Displays setting

### DISCUSSION THREAD

**Discussion by Suyens** [cables-gl/cables_docs#276] [Answers & Questions] [answered]
Hi, Is there any way to set this precision, e.g. x2 ?
![1655300871303](https://user-images.githubusercontent.com/24619296/173843319-085181c5-24ed-4b7e-81d1-a2882378bceb.png)

**Comment by pandrr** [comment, marked as answer]
no (t yet),
but you can use render2texture and render in the resolution you like and then scale it up to full res using fullscreenrectangle.

**Reply by Suyens** [reply]
> no (t yet), but you can use render2texture and render in the resolution you like and then scale it up to full res using fullscreenrectangle.

Thanks.
